TKGame/UI/GameMenu.cs

```diff
@@ -1,15 +1,183 @@
+        private readonly int numRows = 2;
+        private readonly int fontSize = 24;
+        private readonly int playerHpFontSize = 48;
+        private readonly string weaponImageId = "weapon";
+
+
+        public GameMenu()
+        {
+            Initialize();
+            LoadContent();
+        }
+
+        private void Initialize()
+        {
+            grid = new Grid();
+            debugMenu = new DebugMenu();
+            panel = new Panel();
+            panelHsp = new HorizontalStackPanel();
+            playerHealthLabel = new Label();
+            playerGotPowerup = false;
+
+			byte[] ttfData = File.ReadAllBytes(@"Content/Fonts/Retro Gaming.ttf");
+			fontSystem = new FontSystem();
+			fontSystem.AddFont(ttfData);
+		}
+
+        private void LoadContent()
+        {
+            for (int i = 0; i < numCols; i++)
+            {
+                grid.RowsProportions.Add(new Proportion(ProportionType.Auto));
+            }
+            
+            for (int i = 0; i < numRows; i++)
+            {
+                grid.ColumnsProportions.Add(new Proportion(ProportionType.Auto));
+            }
+
+            panel.GridRow = 0;
+            panel.GridColumn = 0;
+            panel.Height = 125;
+            panel.Width = TKGame.ScreenWidth;
+            panel.Margin = new Myra.Graphics2D.Thickness(100, 50, 0, 0);
+            grid.Widgets.Add(panel);
+
+            panelHsp.Spacing = 10;
+            panel.Widgets.Add(panelHsp);
+
+            playerHealthLabel.Font = fontSystem.GetFont(playerHpFontSize);
+            AddWidgetToHeaderPanel(playerHealthLabel, "HP");
+            
+            AddWidgetToHeaderPanel(CreateImageWidget(Art.WeaponTexture, 60, 60, weaponImageId), "Weapon");
+			AddWidgetToHeaderPanel(CreateImageWidget(Art.FireBallTexture, 60, 35), "Q", false);
+			AddWidgetToHeaderPanel(CreateImageWidget(Art.SunBurstTexture, 60, 35, new Rectangle(0, 0, 400, 153)), "E", false);
+			AddWidgetToHeaderPanel(CreateImageWidget(Art.BurningTexture, 60, 60), "Shift", false);
+
+
+			(debugMenu.Container as VerticalStackPanel).GridRow = 1;
+            (debugMenu.Container as VerticalStackPanel).GridColumn = 0;
+            grid.Widgets.Add(debugMenu.Container as VerticalStackPanel);
+
+        }
+
+        private Image CreateImageWidget(Texture2D texture, int width, int height, string name = null)
+        {
+            return CreateImageWidget(texture, width, height, new Rectangle(0, 0, texture.Width, texture.Height), name);
+        }
+
+		private Image CreateImageWidget(Texture2D texture, int width, int height, Rectangle textureBounds, string name = null)
+		{
+			Image image = new Image();
+			image.Renderable = new TextureRegion(texture, textureBounds);
+			image.Width = width;
+			image.Height = height;
+
+            if(name is not null) 
+                image.Id = name;
+
+			return image;
+		}
+
+		private void AddWidgetToHeaderPanel(Widget widget, string text = null, bool isVisible = true)
```

Comment:
tabs

Reply:
definitely want comments to tell the reader what portion of the UI they are reviewer

Reply:
Probably just throw in some regions for this